### PR TITLE
Add firefox/faq (#9490, Fixes #9491, #9498)

### LINKED ISSUE #9491: Epic: faq page
## Description

We want to build a page for firefox faq

url: /firefox/faq/

Link to pitch, brief, or background documents.

---

#### Issues
- [ ] #issue-number
- [ ] #issue-number

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
Adds firefox/faq SEO page.
Includes redirect for [new firefox/browsers/quantum page](#9502).

## Issue / Bugzilla link
#9490
#9491
#9498

## Testing
[Final copy](https://docs.google.com/document/d/1LAdUXUovciI1N4xNN1lNGmWv-6kdXVunvM7arrGyK-E/edit?ts=5f74df55)
[Design](https://github.com/mozilla/bedrock/issues/9497#issuecomment-707772063)

### CODE REVIEW

**Comment by amychurchwell** [comment]
Adding blocked label until Raphael and Chad approve demo.

File: bedrock/firefox/templates/firefox/faq.html
Comment (maxxcrawford) [inline bedrock/firefox/templates/firefox/faq.html:10]: nit: white space

This shows up in the meta description with whitespace at the beginning and end of the entry. 

```suggestion
{% block page_desc %}{{ ftl('page-desc') }}{% endblock %}
```

File: l10n/en/firefox/faq.ftl
Comment (maxxcrawford) [inline l10n/en/firefox/faq.ftl:12]: Is it okay that this line is a duplicate of L9:`page-desc`? As long as the localization team doesn't have to localize the string twice, this can be ignored. 

cc/ @peiying2

File: l10n/en/firefox/faq.ftl
Comment (maxxcrawford) [inline l10n/en/firefox/faq.ftl:32]: Copy (nit): The link should be "here".

image

image

File: l10n/en/firefox/faq.ftl
Comment (maxxcrawford) [inline l10n/en/firefox/faq.ftl:52]: Copy (nit): The link should be "download page". 

image


image

File: l10n/en/firefox/faq.ftl
Comment (maxxcrawford) [inline l10n/en/firefox/faq.ftl:59]: Copy (nit): The link should be "Learn more about the privacy in our products."

image

image

**Review by maxxcrawford — CHANGES_REQUESTED** [review]
There's a visual issue with the footer: 
image

File: bedrock/firefox/templates/firefox/faq.html
Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/faq.html:26]: Thanks Maxx! This comment will be a WONTFIX from me, because the wordmark logo is not the convention we use for [page heros](https://protocol.mozilla.org/patterns/organisms/hero.html#a-firefox-branded-hero-with-image).

File: l10n/en/firefox/faq.ftl
Comment (amychurchwell) [reply l10n/en/firefox/faq.ftl:12]: I will consolidate 👍

File: bedrock/firefox/templates/firefox/faq.html
Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/faq.html:93]: Whoops. That string was missing a `<p>`. Good catch, thank you.

**Comment by amychurchwell** [comment]
Thanks for the thoughtful review, @maxxcrawford! Fixed that failing redirect test, adjusted links and copy accordingly, and made that `<li>` scss selector a little more specific to fix the footer.

I also included some copy changes on `quantum.ftl` [per Peiying's comments.](https://github.com/mozilla-l10n/www-l10n/pull/121#pullrequestreview-513969420)

**Comment by amychurchwell** [comment]
@maxxcrawford structured data has been added and this has been approved to merge if it looks good to you!

**Review by maxxcrawford — APPROVED** [review]

**Pull request merged**
